refinery/core/views.py

```diff
@@ -1015,11 +1015,82 @@ def partial_update(self, request, uuid, format=None):
 
         return Response(serializer.data)
 
-    def is_user_a_group_manager(self, user, group):
+
+class GroupMemberAPIView(APIView):
+    """API endpoint that allows for Group Members to be promoted,
+    demoted or removed"""
+    http_method_names = ['delete', 'post']
+
+    def get_object(self, uuid):
+        try:
+            return ExtendedGroup.objects.get(uuid=uuid)
+        except ExtendedGroup.DoesNotExist as e:
+            logger.error(e)
+            raise Http404
+        except ExtendedGroup.MultipleObjectsReturned as e:
+            logger.error(e)
+            raise APIException("Multiple groups returned for this request.")
+
+    def get_user(self, id):
```

Comment:
Why use Django internal ID instead of UUID like everywhere else?

Reply:
Legacy of previous api. url /group/:uuid/member/:id
I can update it to both use uuid after merging the other two branches into here.

Reply:
Yes, it would be great to fix this.

Reply:
@hackdna Users don't have UUID. User profiles have UUIDs. Simplicity wise, it would make sense to use IDs verses having to grab users through their profiles.

Reply:
I see. It would be great to switch to a custom User model as Two Scoops and [Django docs](https://docs.djangoproject.com/en/1.8/topics/auth/customizing/#specifying-a-custom-user-model) suggest but it's out of the scope of this PR.